javav2/example_code/transcribe/src/main/java/com/amazonaws/transcribestreaming/StreamingRetryApp.java

```diff
@@ -1,119 +1,116 @@
-            } else {
-                this.currentSubscription.cancel();
-                this.currentSubscription = new TranscribeStreamingDemoApp.SubscriptionImpl(s, inputStream);
-            }
-            s.onSubscribe(currentSubscription);
-        }
-    }
-}
-// snippet-end:[transcribe.java-streaming-retry-app]
\ No newline at end of file
+// snippet-sourcedescription:[StreamingRetryApp.java is an application that demonstrates using the Amazon Transcribe retry client.]
+// snippet-service:[transcribe]
+// snippet-keyword:[Java]
+// snippet-sourcesyntax:[java]
+// snippet-keyword:[Amazon Transcribe]
+// snippet-keyword:[Code Sample]
+// snippet-keyword:[TranscribeStreamingAsyncClient]
+// snippet-keyword:[StartStreamTranscriptionResponse]
+// snippet-sourcetype:[snippet]
+// snippet-sourcedate:[2019-01-10]
+// snippet-sourceauthor:[AWS]
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * This file is licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License. A copy of
+ * the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ */
+// snippet-start:[transcribe.java-streaming-retry-app]
+package com.amazonaws.transcribestreaming;
+
+import org.reactivestreams.Publisher;
+import org.reactivestreams.Subscriber;
+import org.reactivestreams.Subscription;
+import software.amazon.awssdk.auth.credentials.EnvironmentVariableCredentialsProvider;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.transcribestreaming.model.AudioStream;
+import software.amazon.awssdk.services.transcribestreaming.model.LanguageCode;
+import software.amazon.awssdk.services.transcribestreaming.model.StartStreamTranscriptionRequest;
+import javax.sound.sampled.LineUnavailableException;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+import java.net.URISyntaxException;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import com.amazonaws.transcribestreaming.TranscribeStreamingDemoApp;
+
+
+public class StreamingRetryApp {
+    private static final String endpoint = "endpoint";
+    private static final Region region = Region.US_EAST_1;
+    private static final int sample_rate = 28800;
+    private static final String encoding = " ";
+    private static final String language = LanguageCode.EN_US.toString();
+
+    public static void main(String args[]) throws URISyntaxException, ExecutionException, InterruptedException, LineUnavailableException, FileNotFoundException {
+        /**
+         * Create Transcribe streaming retry client using AWS credentials.

```

Comment:
Transcribe always requires the "Amazon" in front, so: Amazon Transcribe

Reply:
fixed